src/fidesops/service/connectors/query_config.py

```diff
@@ -410,12 +479,10 @@ def format_fields_for_query(
         return [f'"{field_path.levels[-1]}"' for field_path in field_paths]
 
     def format_clause_for_query(
-        self,
-        field_name: str,
-        operator: str,
+        self, string_path: str, operator: str, operand: str
```

Comment:
Nit: could we leave trailing commas on the end of function args so that black will continue to put them each on newlines?